nncf/quantization/algorithms/smooth_quant/openvino_backend.py

```diff
@@ -170,3 +134,42 @@ def scale_insertion_command(
         source_node: NNCFNode, scale_value: np.ndarray, port_id: int, nodes: List[NNCFNode]
     ) -> OVMultiplyInsertionCommand:
         return OVCommandCreator.multiply_insertion_command(source_node, nodes, port_id, scale_value)
+
+    @staticmethod
+    def get_activation_channel_axis(node: NNCFNode, port_id: int) -> int:
```

Comment:
It seems that the same things made inside of `_get_reduction_shape_and_use_abs_max` in `nncf/nncf/quantization/algorithms/min_max/openvino_backend.py`
Can we merge the same part?

Reply:
We don't do the same thing in `nncf/nncf/quantization/algorithms/min_max/openvino_backend.py`. The algorithms are different. The `channel_axis` for SQ and MinMax is not the same.

Reply:
Got it